logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -26,6 +27,14 @@ public static void start(final Logger logger, final Logger auditor, final String
         schedulerLogger.startInstance(schedulerName);
     }
 
+    public static SchedulerLogger startAsync(final Logger logger, final Logger auditor, final String schedulerName) {
+
+        SchedulerLogger schedulerLogger = new SchedulerLogger();
```

Comment:
Yes, it does. It calls initInstance(), in which we have the line:
    instance.flowContext = FlowContextFactory.serializeNativeFlowContext();